collector/slave_hosts.go

```diff
@@ -0,0 +1,99 @@
+// Scrape heartbeat data.
+
+package collector
+
+import (
+	"database/sql"
+	"github.com/satori/go.uuid"
```

Comment:
We normally put an empty line between Go standard libraries and external libraries.